### PR TITLE
Update issue text for Binary Artifacts policy.

### PR DESCRIPTION
**Pull request by jeffmendoza**
Include reasoning, remediation, and links. Include list of artifacts found.

### CODE REVIEW

**Comment by jeffmendoza** [comment]
cc @brianrussell2

File: pkg/policies/binary/binary.go [hunk 1]
```diff
@@ -172,23 +173,43 @@ func (b Binary) Check(ctx context.Context, c *github.Client, owner,
...
+	logs := convertLogs(l.logs)
+	pass := res.Score >= checker.MaxResultScore
 	var notify string
-	if res.Score < checker.MaxResultScore {
-		notify = fmt.Sprintf("Scorecard Check Binary Artifacts failed: %v\n"+
-			"Please run scorecard directly for details: https://github.com/ossf/scorecard\n",
+	if !pass {
+		notify = fmt.Sprintf(
+			"Scorecard Check Binary Artifacts failed: %v\n"+
+				"Binary Artifacts are an increased security risk in your repository. Binary artifacts cannot be reviewed, allowing possible obsolete or maliciously subverted executables.\n"+
```
Comment (olivekl) [inline pkg/policies/binary/binary.go]: minor nit: "allowing THE INTRODUCTION OF POSSIBLY obsolete or maliciously..."

File: pkg/policies/binary/binary.go [hunk 2]
```diff
@@ -172,23 +173,43 @@ func (b Binary) Check(ctx context.Context, c *github.Client, owner,
...
+				"To remediate, remove the generated executable artifacts from the repository. Build from source where possible.\n"+
+				"For more information see https://github.com/ossf/scorecard/blob/main/docs/checks.md#binary-artifacts. Also, you may run scorecard directly on this repository for more details.\n",
```
Comment (olivekl) [inline pkg/policies/binary/binary.go]: replace "scorecard" with "Security Scorecards" and link if possible?

**Review by olivekl — DISMISSED** [review]
Minor comments, otherwise LGTM!

File: pkg/policies/binary/binary.go (hunk 1 above)
Comment (jeffmendoza) [reply pkg/policies/binary/binary.go]: Fixed

File: pkg/policies/binary/binary.go (hunk 2 above)
Comment (jeffmendoza) [reply pkg/policies/binary/binary.go]: Fixed

File: pkg/policies/binary/binary.go [hunk 3]
```diff
@@ -172,23 +173,43 @@ func (b Binary) Check(ctx context.Context, c *github.Client, owner,
 		return nil, res.Error2
 	}
 
+	logs := convertLogs(l.logs)
+	pass := res.Score >= checker.MaxResultScore
```
Comment (azeemshaikh38) [inline pkg/policies/binary/binary.go:177]: Maybe check the if `len(binaryArtifactsFound) > 0` here? That way, this code will be agnostic to Scorecard's scoring mechanism.

**Review by azeemshaikh38 — APPROVED** [review]

File: pkg/policies/binary/binary.go (hunk 3 above)
Comment (jeffmendoza) [reply pkg/policies/binary/binary.go:177]: Is that available through checker.CheckResult ?

I think we should take a look at this in #21 and #114.

**Pull request merged**